mkmm

```diff
@@ -4,7 +4,8 @@
 # A successor to https://github.com/saber-nyan/kernel-modules-hook
 # Released under the MIT License
 
-_myname="mkmm"
+_myname='mkmm'
+_bakpath="/tmp/mkmm"
```

Comment:
* We must check that this temporary directory belongs to us (and not an adversary).
* We shoud use `$TMPDIR`

Reply:
`TMPDIR` is not set for me, should it be? 

:shrug: https://www.reddit.com/r/archlinux/comments/mmv5e4/arch_does_not_have_tmpdir/

Reply:
and it’s sufficient if ownership of `/tmp/mkmm/$kver` is `root:root`, right?